build-system/tasks/visual-diff/index.js

```diff
@@ -388,6 +389,7 @@ async function generateSnapshots(webpages) {
   // no interactions, and each test that has in interactive tests file should
   // load those tests here.
   for (const webpage of webpages) {
+    webpage.name += argv.esm ? ' (Module)' : ' (Nomodule)';
```

Comment:
Glad this was just a one-line change. (Nice work by @danielrozenberg in writing this file to allow for it!)